README.md

```diff
@@ -1,4 +1,4 @@
-# MyCrypto V4+ (ALPHA - VISIT [V3](https://github.com/MyCryptoHQ/mycrypto.com) for the current site)<br/>Just looking to download? Grab our [latest release](https://github.com/MyCryptoHQ/MyCrypto/releases)
+# MyCrypto Beta (VISIT [MyCrypto.com](https://github.com/MyCryptoHQ/mycrypto.com) for the current site)<br/>Just looking to download? Grab our [latest release](https://github.com/MyCryptoHQ/MyCrypto/releases)
```

Comment:
Nitpicking, but maybe make this text `MyCryptoHQ/mycrypto.com` to make it clear this isn't a link to mycrypto.com, but the github repo.

Reply:
Good point.

Reply:
Fixed